### PR TITLE
Vector capacities update, code cleaned

### PR DESCRIPTION
**Pull request by gchiappe**
- updated values in vector capacities
- removed commented out code
- deployed in demo server

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -102,11 +100,6 @@ pub extern "C" fn _PG_init() {
     info!("ketteQ In-Memory Calendar Cache Extension Loaded (kq_cx)");
 }
 
-#[pg_guard]
-pub extern "C" fn _PG_fini() {
```
Comment (ketteq-neon) [inline src/lib.rs:106]: what would be the reason to remove it?

File: src/lib.rs [hunk 2]
```diff
@@ -20,11 +21,8 @@ AND (table_name = 'calendar' OR table_name = 'calendar_date');"#;
 
 const DEF_Q2_GET_CALENDAR_IDS: &CStr = cr#"SELECT MIN(c.id), MAX(c.id) FROM plan.calendar c"#;
 
-const DEF_Q3_GET_CAL_ENTRY_COUNT: &CStr = cr#"SELECT id, lower(xuid) FROM plan.calendar c ORDER BY id ASC;"#;
-
-// const DEF_Q4_GET_ENTRIES: &CStr = cr#"SELECT cd.calendar_id, cd."date"
-// FROM plan.calendar_date cd
-// ORDER BY cd.calendar_id asc, cd."date" ASC;"#;
+const DEF_Q3_GET_CAL_ENTRY_COUNT: &CStr =
+    cr#"SELECT id, lower(xuid) FROM plan.calendar c ORDER BY id ASC;"#;
```
Comment (ketteq-neon) [inline src/lib.rs]: lets remove lower() here...
so we do not do any conversion and use as is

and remove it in this function
```
unsafe fn **kq_cx_add_days_xuid**(
    input_date: Date,
    interval: i32,
    calendar_xuid: &str,
) -> Option<PgDate> {
    ensure_cache_populated();
    let calendar_xuid = calendar_xuid.to_lowercase();
```

File: src/lib.rs [hunk 3]
```diff
@@ -20,11 +21,8 @@ AND (table_name = 'calendar' OR table_name = 'calendar_date');"#;
 
 const DEF_Q2_GET_CALENDAR_IDS: &CStr = cr#"SELECT MIN(c.id), MAX(c.id) FROM plan.calendar c"#;
 
-const DEF_Q3_GET_CAL_ENTRY_COUNT: &CStr = cr#"SELECT id, lower(xuid) FROM plan.calendar c ORDER BY id ASC;"#;
-
-// const DEF_Q4_GET_ENTRIES: &CStr = cr#"SELECT cd.calendar_id, cd."date"
-// FROM plan.calendar_date cd
-// ORDER BY cd.calendar_id asc, cd."date" ASC;"#;
+const DEF_Q3_GET_CAL_ENTRY_COUNT: &CStr =
+    cr#"SELECT id, xuid FROM plan.calendar c ORDER BY id ASC;"#;
```
Comment (ketteq-neon) [inline src/lib.rs:25]: change query to this one:

```
WITH
	dates AS (
	    SELECT
	        cd.calendar_id,
	        cd."date",
	        ROW_NUMBER() OVER (PARTITION BY cd.calendar_id ORDER BY cd."date" DESC) as row_num
	    FROM
	    	plan.calendar_date cd
	)
SELECT
	calendar_id, "date"
FROM
	dates
WHERE
	row_num <= 5120
ORDER BY
	1, 2
```

File: src/lib.rs [hunk 4]
```diff
@@ -20,11 +21,8 @@ AND (table_name = 'calendar' OR table_name = 'calendar_date');"#;
...
 
 const DEF_Q4_GET_ENTRIES: &CStr = cr#"WITH ranked_dates AS (
     SELECT
```
Comment (ketteq-neon) [inline src/lib.rs:40]: change query to this:
```
WITH
	dates AS (
	    SELECT
	        cd.calendar_id,
	        cd."date",
	        ROW_NUMBER() OVER (PARTITION BY cd.calendar_id ORDER BY cd."date" DESC) as row_num
	    FROM
	    	plan.calendar_date cd
	)
SELECT
	calendar_id, "date"
FROM
	dates
WHERE
	row_num <= 8*1024
ORDER BY
	1, 2
```

File: src/lib.rs [hunk 5]
```diff
@@ -9,9 +9,10 @@ use std::ffi::CStr;
 
 pgrx::pg_module_magic!();
 
-const MAX_CALENDARS: usize = 128;
+const MAX_CALENDARS: usize = 64;
 const MAX_ENTRIES_PER_CALENDAR: usize = 5 * 1024;
```
Comment (ketteq-neon) [inline src/lib.rs]: change this one to 8*1024

File: src/lib.rs [hunk 6]
```diff
@@ -9,9 +9,10 @@ use std::ffi::CStr;
 
 pgrx::pg_module_magic!();
 
-const MAX_CALENDARS: usize = 128;
-const MAX_ENTRIES_PER_CALENDAR: usize = 5 * 1024;
-const CALENDAR_XUID_MAX_LEN: usize = 128;
+const MAX_CALENDARS: usize = 64;
+const MAX_ENTRIES_PER_CALENDAR: usize = 8 * 1024;
+const MAX_PAGES_PER_CALENDAR: usize = 1024;
```
Comment (kq-ng-neon) [inline src/lib.rs]: lets change it back to 512

File: src/lib.rs [hunk 7]
```diff
@@ -20,33 +21,32 @@ AND (table_name = 'calendar' OR table_name = 'calendar_date');"#;
...
+	    	plan.calendar_date cd
+	)
+SELECT
+	calendar_id, "date"
+FROM
+	dates
+WHERE
+	row_num <= 8*1024
+ORDER BY
+	1, 2;"#;
```
Comment (kq-ng-neon) [inline src/lib.rs]: sorry. new query:
```
WITH
	dd AS (
		SELECT 
			(date_trunc('year', date) - INTERVAL '10 Years')::date AS min_date, 
			(date_trunc('year', date) + INTERVAL '12 Years')::date AS max_date
		FROM plan.data_date
	)
SELECT
	calendar_id, "date"
FROM
	plan.calendar_date cd
   	CROSS JOIN dd
WHERE
   	cd.date >= dd.min_date AND cd.date < dd.max_date
ORDER BY
	1, 2
;
```

**Review by kq-ng-neon — APPROVED** [review]

**Review by ketteq-neon — APPROVED** [review]

**Pull request merged**